### ISSUE TITLE
quandl package request

### ISSUE THREAD

**Issue by alen12345** [conda-forge/staged-recipes#5176]
We should add quandl to retrieve important financial data: [https://www.quandl.com/](url)

@jakirkham what do you think?

**Comment by jakirkham** [comment]
Are you referring to the Python package or something else?

**Comment by alen12345** [comment]
python package

**Comment by jakirkham** [comment]
Sure, seems reasonable. Do you want to give it a go?

**Comment by alen12345** [comment]
Yes! It's licensed under MIT, but they have no License file in git. I have already asked to add it; in the meantime, shall we go on without and add it later when they update?

**Comment by jakirkham** [comment]
Sure go for it.

That's unfortunate. Well maybe they will address the issue in the interim. We try to package license files if at all possible and encourage developers to add them. However not everyone adds a license file and it doesn't change our need for the software. So we try to be somewhat pragmatic about this.

**Comment by alen12345** [comment]
I agree with you! I will package it shortly :)

**Comment by alen12345** [comment]
@jakirkham Assign that to me :)

**Comment by alen12345** [comment]
#5186 will be ready soon, the first test has passed :)
